source/_integrations/system_log.markdown

```diff
@@ -9,7 +9,7 @@ ha_domain: system_log
 ha_integration_type: system
 ---
 
-The `system_log` integration stores information about all logged errors and warnings in Home Assistant. To view your logs, navigate to **Settings** -> **System** -> **Logs**. In order to not overload Home Assistant with log data, only the 50 last errors and warnings will be stored. Older entries are automatically discarded from the log. It is possible to change the number of stored log entries using the parameter `max_entries`.
+The `system_log` {% term integration %} stores information about all logged errors and warnings in Home Assistant. To view your logs, navigate to {% my logs title="Settings > System > Logs" %}. In order to not overload Home Assistant with log data, only the 50 last errors and warnings will be stored. Older entries are automatically discarded from the log. It is possible to change the number of stored log entries using the parameter `max_entries`.
```

Comment:
We should keep changes limited to the goal of the PR.

The goal of this PR was to remove the missing google maps example, however, it does a bunch of other things too...

Let's not do that.